### PR TITLE
[ONNX] Enable script tests based on recent exporter changes

### PR DESCRIPTION
**Pull request by shubhambhokare1**

### CODE REVIEW

**Review by justinchuby — APPROVED** [review]
LGTM🚀

**Comment by justinchuby** [comment]
cc @BowenBao for merge approval

**Comment by justinchuby** [comment]
cc @BowenBao for merge approval

**Review by BowenBao — APPROVED** [review]

File: test/onnx/model_defs/word_language_model.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def repackage_hidden(h):
         if isinstance(h, torch.Tensor):
             return h.detach()
         else:
-            return tuple(RNNModel.repackage_hidden(v) for v in h)
```
[the remark this answers is not in the document]
Comment (shubhambhokare1) [reply test/onnx/model_defs/word_language_model.py:69]: This format is unsupported by TorchScript interpreter

[the remark this answers is not in the document]
Comment (justinchuby) [reply test/onnx/model_defs/word_language_model.py:69]: A comment may be helpful since it is not obvious by just looking

**Pull request closed without merging**